content-warehouse/src/main/java/contentwarehouse/v1/CreateRuleSet.java

```diff
@@ -56,8 +56,8 @@ public static void createRuleSet(String projectId, String location)
       throws IOException, InterruptedException, ExecutionException, TimeoutException {
     String projectNumber = getProjectNumber(projectId);
 
-        
-    String endpoint = String.format("%s-contentwarehouse.googleapis.com:443", location);
+    String endpoint = "us".equals(location) ? "contentwarehouse.googleapis.com:443"
```

Comment:
Not sure about the codebase, but do we want to lowercase the location string first and check?

Reply:
Generally, the samples don't include robust validation of inputs. That's for readability purposes -- end-users want to quickly understand the concepts, and for the code to be as copy/pasteable as possible. Will leave this comment open in case other reviewers want to weigh in on this. Thanks!!

Reply:
If readability if focus, I'm not a big fan of ternary operators. What about something like:

```
String endpoint = "contentwarehouse.googleapis.com:443"
if (!"us".equals(location)) {
    endpoint = String.format("%s-%s", location, endpoint);
}
```

(As a side effect that would also remove the duplicate "contentwarehouse.googleapis.com:443" string)